ui/common/css/form/_range.scss

```diff
@@ -1,5 +1,4 @@
 @mixin range-thumb {
-  background-image: linear-gradient(to bottom, $c-metal-top, $c-metal-bottom);
   // https://css-tricks.com/sliding-nightmare-understanding-range-input/
 
   border: 1px solid $c-font-dimmer;
```

Comment:
Best I can tell, the second background rule in this block is also ignored. I think a lot of this is just jank.